style-dictionary/color.config.js

```diff
@@ -0,0 +1,31 @@
+const StyleDictionary = require("style-dictionary");
+const _ = require("lodash");
+const fs = require('fs');
+const colorTokens = require("../packages/uswds-tokens/colors")
+
+const USWDSTokens = StyleDictionary.extend({
+  source: ["packages/uswds-tokens/colors/*.json"],
+  platforms: {
+    "scss/colors": {
```

Comment:
Adding an additional `platform` would not extend other tokens, but instead define a new way to transform the tokens listed in the `source`

As we start to discover the types of transforms we'll need to export our various token types, we'll need to decide if we will have a separate config for each or if we could explore defining different token exports. If we go with the latter it'd probably be a good idea to change this variable to `colorTokens`.

Reply:
So to confirm, we'd need a new const extending style dictionary **per** USWD token?

**Example**
```js
const USWDS_COLORS = StyleDictionary.extend({});
const USWDS_SPACING = StyleDictionary.extend({});
const USWDS_FONTS = StyleDictionary.extend({});
…etc
```

Reply:
If we continue with this pattern, yes.

As we continue to transform tokens, there is a possibility that we'll be able to refine our transformation and be able to have one extend and use relevant filters / JSON files to define each token structure.

Here's an example of how we could change the source to all tokens and use filters and defined "platforms" to separate the transformation.

```js
const USWDSTokens = StyleDictionary.extend({
  // change source path to all tokens
  source: ["packages/uswds-tokens/**/*.json"],
  platforms: {
    "scss/colors": {
      transformGroup: "scss",
      buildPath: "packages/uswds-core/src/styles/tokens/color/",
      files: colorTokens.map((colorToken) => ({
        destination: `_${colorToken}.scss`,
        mapName: `system-color-${colorToken}`,
        format: "custom/format/color-map",
        filter: {
          attributes: {
            category: colorToken
          }
        }
      })),
    // define a separate spacing "platform"
    "scss/spacing": {
      transformGroup: "scss",
      buildPath: "packages/uswds-core/src/styles/tokens/spacing/",
      files: spacingTokens.map((spacingToken) => ({
        destination: `_${spacingToken}.scss`,
        mapName: `system-color-${spacingToken}`,
        format: "custom/format/spacing-map", 
        filter: {
          attributes: {
            category: spacingToken
          }
        }
      })),
    },
```

I'm hoping there's a way we can simplify our custom transformer to pass a "quoted" attribute to define when values are quoted and when they aren't. This would give us more flexibility with our templates and make them more versatile for other token structures.

Reply:
For the sake of brevity, I went ahead and updated the `const` to `USWDS_COLORS` for now.

Resolved in [9feca62](https://github.com/uswds/uswds/pull/5898/commits/9feca62eba7d843fa592342ae072b53996436b30)